src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -10,7 +10,6 @@
 
 public class LottoTicket {
     private static int TICKET_LIMIT = 6;
-    private static int SECOND_PLACE_COUNT = 5;
     private static int TICKET_MIN_NUMBER = 1;
     private static int TICKET_MAX_NUMBER = 45;
     Set<Integer> lottoNumbers;
```

Comment:
접근제어자를 default 로 열어두신 이유가 있을까요? 
```suggestion
    private Set<Integer> lottoNumbers;
```